### PR TITLE
added podspec for repo

### PR DESCRIPTION
**Pull request by HazAT**
Please change the author and repo to your needs if you consider merging it.
I tested it locally and it works.

### CODE REVIEW

**Comment by mmatoszko** [comment]
@Constantine-Fry: Is there any reason why this should not be merged?
I also consider adding carthage support, please let me know if it is a good idea.

File: rebekka-source/Rebekka/StreamOperation.swift [hunk 1]
```diff
@@ -70,6 +70,7 @@ internal class StreamOperation: Operation, NSStreamDelegate {
     }
     
     func finishOperation() {
+        NSThread.sleepForTimeInterval(0.1)
```
Comment (constantine-fry) [inline rebekka-source/Rebekka/StreamOperation.swift:73]: What is it for?

**Comment by constantine-fry** [comment]
Thanks for the PR. I have added Podspec and fixed the problem with errors on download operation.

**Comment by constantine-fry** [comment]
https://github.com/Constantine-Fry/rebekka/commit/d1317799b1ec3031bf528ffd6de748f2a16dbfda
https://github.com/Constantine-Fry/rebekka/commit/11013d386b19e36fece48b642d2b6ee1d90a8630

File: rebekka-source/Rebekka/StreamOperation.swift (hunk 1 above)
Comment (HazAT) [inline rebekka-source/Rebekka/StreamOperation.swift:73]: It was my rather "potato" solution for fixing occasionally crashes when queuing many downloads. The problem was that sometimes to stream still receives/transfers data and `self.currentStream?.close()` cuts the stream.
It sometimes happened that you got a `BAD_EXEC`, but sadly no 100% reproducible. 
I am sure that there must be another solution for this, but it needs further testing.

Comment (constantine-fry) [inline rebekka-source/Rebekka/StreamOperation.swift:73]: Hm, I think I have the same problem. Can't reproduce it myself too. Does it look like this one?

```
1   CFNetwork   NetConnection::dequeue(void*) + 40
2   CFNetwork   CoreStreamBase::_streamInterface_Close() + 92
3   CoreFoundation  _CFStreamClose + 104
```

**Pull request closed without merging**